frontend/components/draft/sidebar.tsx

```
@@ -43,6 +43,7 @@ import { Button } from "../ui/button";
 import { createDropletAnnouncement } from "@/lib/requests/feed";
 
 import { ContentActionButton } from "./metadata/content-action-button";
+import { publishDraftToOriginal } from "@/lib/requests/droplet";
```

ESLint

[error] 'publishDraftToOriginal' is defined but never used.

(@typescript-eslint/no-unused-vars)